### PR TITLE
Add analytics section describing user event data

### PR DESCRIPTION
**Pull request by wongstein**
### Summary

This is the linear card: https://linear.app/insomnia/issue/INS-1287/analyticsdata-docs

The purpose of this PR is to add descriptions of the user event data collected by Insomnia/Kong and how it might be used. 

Current state: Very bad first draft, I'd like comments on the intent of this

### CODE REVIEW

File: docs/insomnia/analytics-collected.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+layout: article-detail
+title:  User Analytics Collected
+category: Security
+category-url: security
+---
+
+When you first download Insomnia, you were asked if you wanted to send analytics data to the Insomnia team about your Insomnia Apps performance and your behavior in the app.  For those who clicked "yes", this article is a description of which analytics are collected and sent.
```
Comment (dimitropoulos) [inline docs/insomnia/analytics-collected.md:8]: should we add anything here about how they can check what their status is in the settings (and how to change/remove them in the settings)?

Comment (wongstein) [reply docs/insomnia/analytics-collected.md:8]: Done

File: docs/insomnia/analytics-collected.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
```
Comment (Guaris) [inline docs/insomnia/analytics-collected.md:8]:
```suggestion
When you first download Insomnia, you were asked if you wanted to send analytics data to the Insomnia team about your Insomnia Apps performance and your behavior in the app. This article is a description of what analytics are collected and sent.
```

File: docs/insomnia/analytics-collected.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+You can edit your preference on sharing analytics data with Insomnia via the Insomnia App Preference Page by scrolling down to the "Network Activity" section and checking or un-checking the box next to "Send Usage Statistics"
```
Comment (Guaris) [inline docs/insomnia/analytics-collected.md:10]:
```suggestion
You can edit your preference on sharing analytics data with Insomnia via the Insomnia app Preference Page by scrolling down to the **Network Activity** section and checking or unchecking the box next to **Send Usage Statistics**
```

File: docs/insomnia/analytics-collected.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+# Error Data Collection
+When you opt-in to collection of analytics, the Insomnia app will send anonymized action event data to Kong that may be later used to evaluate user behavior in the purpose of guiding product decisions.
```
Comment (Guaris) [inline docs/insomnia/analytics-collected.md:13]:
```suggestion
When you opt-in to the collection of analytics, the Insomnia app will send anonymized action event data to Kong that may later be used to evaluate user behavior for the purpose of guiding product decisions.
```

File: docs/insomnia/analytics-collected.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+Here is the format of the JSON data body of the event sent:
```
Comment (Guaris) [inline docs/insomnia/analytics-collected.md:15]:
```suggestion
This is the format of the JSON data body for a sent event:
```

**Review by Guaris — CHANGES_REQUESTED** [review]

**Review by Guaris — APPROVED** [review]

**Comment by Guaris** [comment]
LGTM!

**Pull request merged**